.env.example

```diff
@@ -10,3 +10,6 @@ MCC_CODE=302
 
 TEST_MODE=false
 MOCK_SERVER=false
+DOWNLOAD_TIMEOUT_SECONDS=120
+MAXIMUM_DOWNLOAD_SIZE_KB=10000
```

Comment:
This is roughly 10MB. I download last 14 days keys from server. It's 13KB for 635 keys. So, 10MB will be roughly 400k keys which is more than enough, I think. 👍